streaming/src/main/scala/org/apache/spark/streaming/scheduler/ReceiverTracker.scala

```diff
@@ -18,6 +18,8 @@
 package org.apache.spark.streaming.scheduler
 
 
+import org.apache.spark.streaming.util.TimeoutUtils
```

Comment:
Minor style-nit, but this import should be grouped with the other Spark ones down near line 31.